tests/test_contigs_list_contaminants.py

```diff
@@ -10,7 +10,8 @@ def test_1_loomba(location):
     args = utils.Args()
     args.genome = utils.relative_file("demo/genomes/LoombaR_2017__SID1050_bax__bin.11.fa.gz")
     args.genome_sig = utils.relative_file("tests/test-data/loomba/LoombaR_2017__SID1050_bax__bin.11.fa.gz.sig")
-    args.matches_sig = utils.relative_file("tests/test-data/loomba/LoombaR_2017__SID1050_bax__bin.11.fa.gz.matches.abund.zip")
```

Comment:
was this here to see if this would work with signatures that held abundances? Does this matter anymore?

Reply:
I actually don't know why I had abundances there.

Reply:
I re-added this in 7419a3f189, with an appropriate name. (I also checked that it was testing the right bit of code!)

Thanks for catching this - generally I don't like removing tests!